Documentation/cmdref/cilium-agent.md

```diff
@@ -72,6 +72,7 @@ cilium-agent [flags]
       --http-retry-timeout uint                    Time after which a forwarded but uncompleted request is retried (connection failures are retried immediately); defaults to 0 (never)
       --identity-change-grace-period duration      Time to wait before using new identity on endpoint identity change (default 5s)
       --install-iptables-rules                     Install base iptables rules for cilium to mainly interact with kube-proxy (and masquerading) (default true)
+      --ipam string                                Backend to use for IPAM (default "hostscope")
```

Comment:
Are you suggesting to rename `"hostscope"` to `"cilium"`? I dont' care that much about how the option is called but `"cilium"` seems very unspecific.

Reply:
@tgraf yes. Otherwise it might be confusing for users as the "hostscope" name is extremely similar to the "host-local" cni plugin and it might give the impression that Cilium requires this "host-local" cni plugin to run.

Reply:
What name do you propose? `"cilium"` is not meaningful.

Reply:
@tgraf Why not? Hmmm why not having `""` as default and write that "it defaults to internal IPAM if not specified."

Reply:
That I'm fine with. I thought you meant to rename it to `"cilium"`